content/docs/user-guide/how-to/add-output-to-stage.md

```diff
@@ -0,0 +1,69 @@
+# Add Output to Stage
+
+There are situations where you might have forgotten to specify one of the
+<abbr>outputs</abbr> for a stage while creating it, and then later want to add
+an output to that stage.
+
+Running a stage to specify additional output is an expensive step, as it
+requires time and resources to execute the command. Follow the steps listed
+below to add an output to the stage without running it again.
+
+Let's first create a stage for the example <abbr>project</abbr>:
+
+```dvc
+$ dvc run -n prepare \
+          -d src/prepare.py \
+          -o data/train \
+          python src/prepare.py
+```
+
+You can check stage entry in `dvc.yaml`:
+
+```dvc
+$ cat dvc.yaml
+stages:
+  prepare:
+    cmd: python src/prepare.py
+    deps:
+    - src/prepare.py
+    outs:
+    - data/train
+```
+
+Now to add an output to the `prepare` stage, you need to use `dvc run` with the
+`-f` and `--no-exec` options:
+
+```dvc
+$ dvc run -f --no-exec \
+          -n prepare
+          -d src/prepare.py \
+          -o data/train \
+          -o data/test \
+          python src/prepare.py
+```
+
+The `-f` option is used to overwrite an existing stage in the `dvc.yaml`, and
+the `--no-exec` option is used to create or update a stage without executing the
+command defined in it.
+
+Let's check `dvc.yaml` again:
+
+```dvc
+$ cat dvc.yaml
+stages:
+  prepare:
+    ...
+    outs:
```

Comment:
I feel like we could compress the whole How To into a single code block that shows the "diff" between dvc.yaml before and after the new output is added. See https://dvc.org/doc/start/data-pipelines (expand the first **Expand to see what happens under the hood.** to see the block I mean):

![image](https://user-images.githubusercontent.com/1477535/95241633-a6a11300-07d3-11eb-86b5-f6a97bdb9724.png)

Reply:
Same for the `run` command notes, it could be a single block quote after the dvc.yaml block with both run commands one after the other basically. Let's see what you can come up with 🙂

Reply:
I have added this "diff" part but it isn't showing green color in my local environment.
Also as you said I moved the `dvc run` code to a block quote. But I have only kept the second one (with `-f` and `--no-exec`) as this how-to is about what happens after creating a stage. That is, the user might have already created the stage so I think the `dvc run` code showing creating the stage won't be required here. The other reason being that the block quote became too long.

Reply:
> added this "diff" part but it isn't showing green color

It should. Did you check the example from where I took a screenshot to see what the code is? That MD code formatter is `git`.

> moved the dvc run code to a block quote

Goodm thanks

> the user might have already created the stage

Yep, not might: that's exactly the situation we're writing about 🙂